tests/test_main.py

```diff
@@ -14,7 +14,6 @@
 from app.players.elves.elf_ranger import ElfRanger
 from app.players.player import Player
 
-
```

Comment:
no need to change something in tests

Reply:
This happened accidentally.  When I reordered my project into packages PyCharm changed all imports in my files by itself, and in tests too). I had spent all day to find this and resolve why my code doesn't work. So I just restored imports in test-main.py from origin file again.
All imports in tests looked like:
from app import ElfRanger
from app import Player